pkgs/development/compilers/microscheme/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchgit, vim }:
+
+stdenv.mkDerivation rec {
+  name = "microscheme-${version}";
+  version = "2015-02-04";
+
+  # externalize url/rev/sha256 to permit easier override
+  rev = "64dab5dff1a1fb54070bfc9a6d1f1be06cd9ad19";
+  sha256 = "0i3n7vwfbqsf95cif3qjq5hj6npnpwaiajdifnjlrn9jzki8lg40";
+  url = https://github.com/ryansuchocki/microscheme.git;
+
+  src = fetchgit {
+    inherit rev;
+    inherit sha256;
+    inherit url;
+  };
+
+  buildInputs = [ vim ];
+
+  buildPhase = ''
+    make hexify
+    make build
```

Comment:
If "make" is enough, you don't need custom buildPhase at all; the standard buildPhase does "make" for you :-)